OpencartTest/src/com/opencart/pages/account/EditAdressPage.java

```diff
@@ -2,17 +2,333 @@
+
+    public void clickDefaultAddresRadioB() {
+        defaultAddresRadioB.click();
+    }
+
+    //continueButton
+    public void clickContinueSubmitButton() {
+        continueSubmitButton.click();
+    }
+    //FUNCTIONAL
+
+
+    //firstnameInputField
+    public void fillInputFirstname(String firstname) {
+        clickFirstnameInputField();
+        clearFirstnameInputField();
+        setFirstnameInputField(firstname);
+    }
+
+    //firstnameInputField
+    public void fillInputLastname(String lastname) {
+        clickLastnameInputField();
+        clearLastnameInputField();
+        setLastnameInputField(lastname);
+    }
+
+    //addressinputField
+    public void fillAdressfield(String addres) {
+        clickAddresInputField();
+        clearAddresInputField();
+        setAddresInputField(addres);
+    }
+
+    //firstnameInputField
+    public void fillInputCity(String city) {
+        clickCityInputField();
+        clearCityInputField();
+        setCityInputField(city);
+    }
+
+    //Alerts
+    public boolean succesFirstNameAlert() {
+        if (firstNameAlert.getText().equalsIgnoreCase("First Name must be between 1 and 32 characters!")) {
+            return firstNameAlert.isDisplayed();
+        } else return false;
+    }
+
+    public boolean succesLastNameAlert() {
+        if (lastNameAlert.getText().equalsIgnoreCase("Last Name must be between 1 and 32 characters!")) {
+            return lastNameAlert.isDisplayed();
+        } else return false;
+    }
+
+    public boolean succesAddressAlert() {
+        if (addressAlert.getText().equalsIgnoreCase("Address must be between 3 and 128 characters!")) {
+            return addressAlert.isDisplayed();
+        } else return false;
+    }
+
+    public boolean succesCityAlert() {
+        if (cityAlert.getText().equalsIgnoreCase("City must be between 2 and 128 characters!")) {
+            return firstNameAlert.isDisplayed();
+        } else return false;
+    }
+
+    public boolean succesCountryAlert() {
+        if (countryAlert.getText().equalsIgnoreCase("Please select a country!")) {
+            return countryAlert.isDisplayed();
+        } else return false;
+    }
+
+    public boolean succesRegionAlert() {
+        if (regionAlert.getText().equalsIgnoreCase("Please select a region / state!")) {
+            return regionAlert.isDisplayed();
+        } else return false;
+    }
+
+    public boolean allMandatoryAlerts() {
+        if (succesAddressAlert() && succesFirstNameAlert() && succesLastNameAlert() && succesCityAlert()) {
+            return true;
+        } else return false;
+    }
+
+    public boolean firstLastNameAlerts() {
+        if (succesFirstNameAlert() && succesLastNameAlert()) {
+            return true;
+        } else return false;
+    }
+
+    //Busines logic
+    public AddressBookPage register(String FIRST_NAME, String LAST_NAME, String ADDRESS, String CITY, String COUNTRY, String REGION) {
+        fillInputFirstname(FIRST_NAME);
```

Comment:
please read java convention
you can't use such format for method params

Reply:
> please read java convention
> you can't use such format for method params
Corected)